core/src/toga/widgets/base.py

```diff
@@ -9,22 +9,25 @@
 
 from toga.platform import get_platform_factory
 from toga.style import Pack, TogaApplicator
+from toga.style.mixin import style_mixin
 
 if TYPE_CHECKING:
     from toga.app import App
     from toga.window import Window
 
 StyleT = TypeVar("StyleT", bound=BaseStyle)
+PackMixin = style_mixin(Pack)
 
 
-class Widget(Node):
+class Widget(Node, PackMixin):
```

Comment:
I put the style properties into a separate base class to avoid them cluttering the documentation.